src/Functions/seriesPeriodDetect.cpp

```diff
@@ -0,0 +1,124 @@
+#pragma clang diagnostic push
```

Comment:
Function documentation (subdirectory "docs/en/sql-reference/functions") is missing.